extensions/sql-database-projects/src/test/netCoreTool.test.ts

```diff
@@ -48,13 +47,19 @@ describe('NetCoreTool: Net core tests', function (): void {
 
 		if (os.platform() === 'win32') {
 			// check that path should start with c:\program files
-			let result = isNullOrUndefined(netcoreTool.netcoreInstallLocation) || netcoreTool.netcoreInstallLocation.toLowerCase().startsWith('c:\\program files');
+			let result = netcoreTool.netcoreInstallLocation === undefined || netcoreTool.netcoreInstallLocation === null || netcoreTool.netcoreInstallLocation.toLowerCase().startsWith('c:\\program files');
 			should(result).true('dotnet is either not present or in programfiles by default');
 		}
 
-		if (os.platform() === 'linux' || os.platform() === 'darwin') {
+		if (os.platform() === 'linux'){
+			//check that path should start with /usr/share
+			let result = netcoreTool.netcoreInstallLocation === undefined || netcoreTool.netcoreInstallLocation === null || netcoreTool.netcoreInstallLocation.toLowerCase().startsWith(NetCoreLinuxDefaultPath);
```

Comment:
any reason to only check `startsWith()` rather than checking if the install location equals the expected default path?

Reply:
Nope, I was just following the pattern. Updated it to match the exact install location.

Reply:
also update the windows one :)

Reply:
For windows, dotnet is generally present in C:\\Program Files, but it is possible to have it under C:\\Program Files(x86). So rather than checking for both conditions, I think it should be ok to keep it as-is for Windows. Thoughts?

Reply:
ah that's right. Ok makes sense to keep it as is then